src/lib/viewers/doc/docAssets.js

```diff
@@ -1,13 +1,18 @@
-import Browser from '../../Browser';
-import { DOC_LEGACY_STATIC_ASSETS_VERSION, DOC_STATIC_ASSETS_VERSION } from '../../constants';
+import { DOC_STATIC_ASSETS_VERSION } from '../../constants';
 
-const STATIC_VERSION = Browser.isIE() ? DOC_LEGACY_STATIC_ASSETS_VERSION : DOC_STATIC_ASSETS_VERSION;
-const STATIC_URI = `third-party/doc/${STATIC_VERSION}`;
+const STATIC_URI = `third-party/doc/${DOC_STATIC_ASSETS_VERSION}`;
+const versionParts = DOC_STATIC_ASSETS_VERSION.split('.').map(part => parseInt(part, 10));
+const JS_EXTENSTION =
```

Comment:
What if we set JS_EXTENSTION to 'mjs' by default and just add a comment saying why? If we had to revert to a previous version of pdf.js we would have to test and push out a new version anyway and tests would not pass if this value was not set correctly e.g. back to .js . This just eliminates some of the logic and simplifies the code a bit. Thoughts?

Reply:
Yeah I like this approach. I don't love the version check either because the version numbers are kind of arbitrary. I'll just remove the `JS_EXTENSION` variable completely